packages/react/src/components/Table/tableReducer.test.jsx

```diff
@@ -473,6 +473,50 @@ describe('table reducer', () => {
       // is Loading should be set false and rowCount should be correct
       expect(tableWithSortedData.view.table.loadingState.isLoading).toEqual(false);
       expect(tableWithSortedData.view.table.loadingState.rowCount).toEqual(0);
+
+      // load more
+      const initialStateLoadMore = {
```

Comment:
Wouldn't we wanna see a test where the the REGISTER_TABLE is called with e.g. 3 ids in view.table.loadingMoreIds and with data that indicates that 2 rows have gotten more children added. Then have the `expect` verify that the loadingMoreIds only contains the id of the row that didn't get additional children added?